src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -132,7 +133,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
                     $this->hardCopy($originDir, $targetDir);
                 }
             }
+            if (is_dir($targetDir)) {
+                array_push($validAssetDir, $targetDir);
+            }
+        }
+        // Check in $bundlesDir, if all links/folder still have an existing Bundle
+        if ($dir = opendir($bundlesDir)) {
```

Comment:
opendir() 😆 it's at least 10 years ago since a last used that function (you make me feel old 😢 😉).